web/src/pages/Analytics.tsx

```diff
@@ -0,0 +1,376 @@
+import { ChevronDownIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Container,
+  Flex,
+  Grid,
+  GridItem,
+  Heading,
+  Icon,
+  Progress,
+  Stack,
+  Stat,
+  StatHelpText,
+  StatLabel,
+  StatNumber,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+  useColorModeValue,
+  useMediaQuery,
+} from "@chakra-ui/react";
+import { format } from "d3-format";
+import { interpolateBuPu } from "d3-scale-chromatic";
+import { Bounds } from "pigeon-maps";
+import { useState } from "react";
+import { BsGearFill } from "react-icons/bs";
+import { HiBell, HiOfficeBuilding, HiRefresh } from "react-icons/hi";
+import { useRecoilValue } from "recoil";
+import useSWR from "swr";
+
+import { ConnectToSingleStoreButton } from "@/components/ConnectToSinglestoreButton";
+import { Loader } from "@/components/customcomponents/loader/Loader";
+import { EnableSimulatorButton } from "@/components/EnableSimulatorButton";
+import { Heatmap } from "@/components/HeatMap";
+import { SetupDatabaseButton } from "@/components/SetupDatabaseButton";
+import {
+  CustomerMetrics,
+  customerMetrics,
+  estimatedRowCountObj,
+  overallConversionRate,
+  ZoneMetrics,
+  zoneMetrics,
+} from "@/data/queries";
+import { connectionConfig, simulatorEnabled } from "@/data/recoil";
+import { useConnectionState } from "@/view/hooks/hooks";
+import { useSimulationMonitor } from "@/view/hooks/useSimulationMonitor";
+import { useSimulator } from "@/view/hooks/useSimulator";
+
+const formatPct = format(",.2%");
+const formatStat = format(".4~s");
+
+const NotificationZoneMap = () => {
+  const [isSmallScreen] = useMediaQuery("(max-width: 640px)");
+
+  return (
+    <Flex gap={5} direction={isSmallScreen ? "column" : "row"}>
```

Comment:
do we need this `isSmallScreen`? Can't we use Flex `wrap`?

Reply:
the way components should be arranged is different from big screen to small screen

flexWrap would have been useful if the arrangement style is same